tests/e2e/vmservice_late_binding.go

```diff
@@ -269,4 +309,563 @@ var _ bool = ginkgo.Describe("[vmsvc] VM-Service-VM-LateBinding", func() {
+
+		ginkgo.By("Create vm service vm")
+		vm1 := createVmServiceVmWithPvcs(
+			ctx, vmopC, namespace, vmClass, []*v1.PersistentVolumeClaim{pvc1, pvc2, pvc3, pvc4}, vmi,
+			wffsStoragePolicyName, secretName)
+
+		ginkgo.By("Creating loadbalancing service for ssh with the VM")
+		_ = createService4Vm(ctx, vmopC, namespace, vm1.Name)
+
+		ginkgo.By("Wait for VMs to come up and get an IP")
+		vmIp1, err := waitNgetVmsvcVmIp(ctx, vmopC, namespace, vm1.Name)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Waiting PVC to be in bound state")
+		_, err = fpv.WaitForPVClaimBoundPhase(ctx, client, []*v1.PersistentVolumeClaim{pvc1, pvc2, pvc3,
+			pvc4}, pollTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("get VM storage")
+		vmQuotaUsed := getVMStorageData(ctx, vmopC, namespace, vm1.Name)
+		framework.Logf("vmQuotaUsed : %s", vmQuotaUsed)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify PVCs are accessible to the VM")
+		ginkgo.By("Write some IO to the CSI volumes and read it back from them and verify the data integrity")
+		//vm1, err = getVmsvcVM(ctx, vmopC, vm1.Namespace, vm1.Name) // refresh vm info
+		instanceKey := ctlrclient.ObjectKey{Name: vm1.Name, Namespace: namespace}
+		vm1 = &vmopv1.VirtualMachine{}
+		err = vmopC.Get(ctx, instanceKey, vm1)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		for i, vol := range vm1.Status.Volumes {
+			if vol.Name == pvc1.Name || vol.Name == pvc2.Name || vol.Name == pvc3.Name || vol.Name == pvc4.Name {
+				volFolder := formatNVerifyPvcIsAccessible(vol.DiskUuid, i+1, vmIp1)
+				verifyDataIntegrityOnVmDisk(vmIp1, volFolder)
+			}
+
+		}
+
+		vm1v4, err := getVmsvcVM4(ctx, vmopC, vm1.Namespace, vm1.Name) // refresh vm info
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		//Get zone details of VM
+		zone, err := getVMzone(ctx, vm1v4)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		//Verify Annotations and Zonal details  on PVC
+		pvc1, err = client.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvc1.Name, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = validateAnnotationOnPVC(pvc1, selectedNodeIsZone, "true")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		pvc2, err = client.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvc2.Name, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = validateAnnotationOnPVC(pvc2, selectedNodeIsZone, "true")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		err = validateAnnotationOnPVC(pvc1, selectedNodeAnnotationOnPVC, zone)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		err = validateAnnotationOnPVC(pvc2, selectedNodeAnnotationOnPVC, zone)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		err = validateAnnotationOnPVC(pvc3, selectedNodeAnnotationOnPVC, zone)
+		gomega.Expect(err).To(gomega.HaveOccurred())
+
+		// ginkgo.By("Validate StorageQuotaDetails after creating the workloads")
```

Comment:
remove commented code

Reply:
Quota validation is requited , Since it is moved to bytes I have commented. In next PR  I will fix this